test/testcafe/spec/UnlockAccountChallengeIdentifierFirst_spec.js

```diff
@@ -124,13 +173,45 @@ test.requestHooks(identifyLockedUserMock)('should render custom Unlock account l
   await t.expect(identityPage.getCustomUnlockAccountLinkUrl(customUnlockLinkText)).eql('http://unlockaccount');
 });
 
+// Legacy faked identifier-first flow only existed for gen3
+test.meta('gen2', false).requestHooks(legacyIdentifyLockedUserMock)('should show unlock account authenticator selection list when user has more than one authenticator (legacy faked flow)', async t => {
+  const identityPage = await setup(t);
+  await checkA11y(t);
+  await identityPage.clickUnlockAccountLink();
+
+  const selectFactorPage = new SelectFactorPageObject(t);
+  await t.expect(selectFactorPage.getFormTitle()).contains('Unlock account');
+  await selectFactorPage.fillIdentifierField('username');
+
+  await t.expect(selectFactorPage.getNextButton().exists).eql(true);
+  await selectFactorPage.goToNextPage();
+
+  await t.expect(selectFactorPage.getFactorsCount()).eql(2);
+  await selectFactorPage.selectFactorByIndex(0);
+
+  const challengeEmailPageObject = new ChallengeEmailPageObject(t);
+  await t.expect(challengeEmailPageObject.getFormTitle()).eql('Verify with your email');
+  await challengeEmailPageObject.clickEnterCodeLink();
+  await challengeEmailPageObject.verifyFactor('credentials.passcode', '12345');
+  await challengeEmailPageObject.clickVerifyButton();
+
+  const successPage = new TerminalPageObject(t);
+  await t.expect(successPage.getFormTitle()).eql('Account successfully unlocked!');
+  await t.expect(successPage.doesTextExist('You can log in using your existing username and password.')).eql(true);
+  const gobackLinkExists = await successPage.goBackLinkExistsV2();
+  await t.expect(gobackLinkExists).eql(false);
+  const signoutLinkExists = await successPage.signoutLinkExists();
+  await t.expect(signoutLinkExists).eql(true);
+});
+
+
 test.requestHooks(identifyLockedUserMock)('should show unlock account authenticator selection list when user has more than one authenticator', async t => {
   const identityPage = await setup(t);
   await checkA11y(t);
   await identityPage.clickUnlockAccountLink();
 
   const selectFactorPage = new SelectFactorPageObject(t);
-  await t.expect(selectFactorPage.getFormTitle()).eql('Unlock account?');
+  await t.expect(selectFactorPage.getFormTitle()).contains('Unlock account');
```

Comment:
Why did this change to `contains`?

Reply:
Ah, so I changed those to support that gen2 uses the "Unlock account" string w/o a question mark but the faked gen3 flow was implemented with the "Unlock account?" string. That being said, it should actually be fine (and better) if I update the string for the faked gen3 flow to also use the string w/o a question mark. I'll update this as well, nice call out